system/metrics/sinks/elastic/elasticSink.cpp

```diff
@@ -193,3 +311,17 @@ void ElasticMetricSink::collectingHasStopped()
 
 }
 
+
+bool ElasticMetricSink::validateIndex()
+{
+    std::string endpoint;
+    endpoint.append("/").append(indexName.str());
+    auto res = pClient->Get(endpoint.c_str(), elasticHeaders);
+
+    if (res->status != 200)
```

Comment:
is res an http code only? If there's a message as well, it should be reported in the log message as well.